package.json

```diff
@@ -85,7 +85,7 @@
     "webpack-stream": "^4.0.0"
   },
   "devEngines": {
```

Comment:
```suggestion
  "resolutions": {
    "gulp/**/natives": "1.1.3"
  },
  "devEngines": {
```

Reply:
Please use Yarn's [selective dependency version resolution](https://yarnpkg.com/en/docs/selective-version-resolutions) instead of editing the lockfile directly. You'll have to run `yarn install` after making this change and commit the updated lockfile. Make sure to use a recent version of yarn.

Reply:
Thank you!

Reply:
Thank you!

Reply:
Thank you!

Reply:
Thank you!